### DISCUSSION TITLE
Welcome to docFiller Discussions!

### DISCUSSION THREAD

**Discussion by rootCircle** [rootCircle/symmetrical-octo-spork#78] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KeethFrancMalcolm** [comment]
I have a question. I have ollama installed and I do not know how to connect that with docfiller. Any help?

**Reply by rootCircle** [reply]
Hey great question, in order to use Ollama with docfiller, currently docfiller expects certain models like qwen:4b via ollama, this is completely configurable at code level, but still not at user level. (PRs welcome)

When running Ollama, make sure qwen3:4b is installed and run ollama using `OLLAMA_ORIGINS=* HOME=/var/lib/ollama OLLAMA_MODELS=/var/lib/ollama ollama serve` or set `OLLAMA_ORIGINS=* && ollama serve` in Linux or Windows CMD respectively!

If you face any issues let me know!
The support for Ollama is not smooth as compared to other LLMs hosted on cloud like chatgpt. The reason is consumer hardware might be slow and many a time responses times out! (Known issue #35)
